.github/workflows/code-checks.yml

```diff
@@ -18,4 +18,7 @@ jobs:
         run: |
           grep mypy requirements/test.txt | xargs -0 pip install
```

Comment:
This is not necessary now

```suggestion
```

Reply:
isn't it since we are not installing requirements/test which is where mypy is located?